pages/security.md

```diff
@@ -190,6 +190,84 @@ The Okta developer blog also has some ❤️ for Micronaut and Quarkus:
 - [Build a Secure Micronaut and Angular App with JHipster](https://developer.okta.com/blog/2020/08/17/micronaut-jhipster-heroku)
 - [Fast Java Made Easy with Quarkus and JHipster](https://developer.okta.com/blog/2021/03/08/jhipster-quarkus-oidc)
 
+### Auth0
+
+If you'd like to use [`Auth0`](https://auth0.com/) instead of `Keycloak`, you can follow the below configuration steps:
+
+#### Create an OIDC App using Auth0 Admin Dashboard
+- Create a free developer account at <https://auth0.com/signup>. After successful sign-up, your account shall be associated with a unique domain like `dev-xxx.us.auth0.com`
+- Create a new application of type `Regular Web Applications`. Switch to the `Settings` tab, and configure your application settings like:
+    - Allowed Callback URLs: `http://localhost:8080/login/oauth2/code/oidc`
+    - Allowed Logout URLs: `http://localhost:8080/`
+- Navigate to **User Management** > **Roles** and create new roles named `ROLE_ADMIN`, and `ROLE_USER`.
+- Navigate to **User Management** > **Users** and create a new user account. Click on the **Role** tab to assign roles to the newly created user account.
+- Navigate to **Auth Pipeline** > **Rules** and create a new Rule. Choose `Empty rule` template. Provide a meaningful name like `JHipster claims` and replace `Script` content with the following and Save.
+```javascript
+function (user, context, callback) {
+  user.preferred_username = user.email;
+  const roles = (context.authorization || {}).roles;
+  function prepareCustomClaimKey(claim) {
+    return `https://www.jhipster.tech/${claim}`;
+  }
+  const rolesClaim = prepareCustomClaimKey('roles');
+  if(context.idToken) {
+  	context.idToken[rolesClaim] = roles;
+  }
+  if(context.accessToken) {
+  	context.accessToken[rolesClaim] = roles;
+  }
```

Comment:
These if statements should have a space after `if` and indent only 2 spaces in their body.

Reply:
should be good now